include/ruby/io.h

```diff
@@ -153,6 +153,7 @@ void rb_io_set_nonblock(rb_io_t *fptr);
 int rb_io_extract_encoding_option(VALUE opt, rb_encoding **enc_p, rb_encoding **enc2_p, int *fmode_p);
 void rb_io_extract_modeenc(VALUE *vmode_p, VALUE *vperm_p, VALUE opthash, int *oflags_p, int *fmode_p, rb_io_enc_t *convconfig_p);
 ssize_t rb_io_bufwrite(VALUE io, const void *buf, size_t size);
+void rb_io_set_encoding_internal(VALUE io, VALUE ext_enc, VALUE int_enc);
```

Comment:
Could it just be declared in `internal.h` ?

Reply:
Yes, that would work too

Reply:
Handled in https://github.com/ruby/ruby/pull/2829/commits/21943301bacefaaae599f2d27e55a5a246767026

Reply:
Sorry, https://github.com/ruby/ruby/pull/2829/commits/236976b7d339309d18387bd83c276f7e35c19416 (had to force push, missed an entry in `common.mk`)